src/extensions/amazon-sales/release-notes/index.md

```diff
@@ -21,6 +21,32 @@ The release notes include:
 
 See [Upcoming Releases](https://devdocs.magento.com/release/) for versioning, support, and compatibility.
 
+### v4.2.0
+
+Amazon Sales Channel 4.2.0 is compatible with versions 2.3.x but only supported in versions 2.4.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. If you have a previous Amazon Sales Channel version installed and attempt to update your Magento to version 2.4.0, you will be prompted to update the extension before you can complete the Magento update.
+
+This version of Amazon Sales Channel includes a new feature along with improvements and fixes.
+
+-  {:.new}Amazon Sales Channel has been enhanced to accept text-based address data and match it to standardized address formats, including city, state, and zip code. This enables order and shipping data to synchronize with Amazon without address errors. Example: Shopper inputs the city, state, zip code as `Escondido, californiA 92025-1501`. Amazon Sales Channel imports and matches the data to the standard format as `Escondido, CA 92025`, and then syncs it back to Amazon in this standardized format.
+
+-  {:.new}Support for PHP 7.4 has been added.
+
+-  {:.fix}<!--CHAN-4334-->The extension has been updated for support with Magento 2.4.x. Previous versions may be compatible with Magento 2.4.x, but are not supported. See [Upcoming releases](https://devdocs.magento.com/release/) for version compatibility. Amazon Sales Channel must be updated to 4.2.0 before the Magento 2.4.0 update can be completed.
+
+-  {:.fix}<!--CHAN-4431-->Corrected an error causing an "Access Denied" error for UK customers.
```

Comment:
Two errors? Is there another way to word this?

Reply:
corrected: issue caused an error